components/esp_websocket_client/Kconfig

```diff
@@ -0,0 +1,9 @@
+menu "ESP WebSocket client"
+
+    config ESP_WS_CLIENT_ENABLE_DYNAMIC_BUFFER
+        bool "Enable websocket client dynamic buffer for tx/rx"
+        default n
+        help
+            This option will enable dynamic buffer malloc for tx/rx, it can save about tx and rx memory when no websocket data send and receive.
```

Comment:
I suggest that we add that the buffer is reallocated for each use, since this is relevant to our context.

Reply:
Sorry, I didn't understand what you meant, can you share your thoughts in detail? Do you mean make tx/rx buffer size seperate?

Reply:
No, I mean to clarify what "dynamic buffer" mean on operation.

Reply:
Ok, i have changed it. Please help review it again. Thanks.